tests/test_dcrModel.py

```diff
@@ -359,39 +346,36 @@ def testRotationFlipped(self):
 
     def testConditionDcrModelNoChange(self):
         """Conditioning should not change the model if it equals the reference.
-
-        This additionally tests that the variance and mask planes do not change.
         """
-        dcrModels = DcrModel(modelImages=self.makeTestImages())
+        modelImages = self.makeTestImages()
+        dcrModels = DcrModel(modelImages=modelImages, mask=self.mask)
         newModels = [model.clone() for model in dcrModels]
         dcrModels.conditionDcrModel(newModels, self.bbox, gain=1.)
         for refModel, newModel in zip(dcrModels, newModels):
-            self.assertMaskedImagesEqual(refModel, newModel)
+            self.assertFloatsAlmostEqual(refModel.array, newModel.array, rtol=1e-4, atol=1e-4)
```

Comment:
That's a surprisingly low tolerance: why so loose?

Reply:
I put the loose tolerances in while tracking down a bug, but they're not needed now. I'll remove them.